### DISCUSSION TITLE
How to pause automatic synchronization in version 5.0

### DISCUSSION THREAD

**Discussion by visiLi** [floccusaddon/floccus#1456] [Q&A] [answered]
My Firefox and Edge browsers don't need to sync automatically, where to set off

**Comment by marcelklehr** [comment, marked as answer]
Make sure you have at least v5.0.2, then check the profile options under "Sync behavior", there should be the Auto-sync option

**Reply by visiLi** [reply]
OK, the problem has been solved.

**Reply by visiLi** [reply]
5.02 Google, browser, settings, whether or not to turn on automatic synchronization, the slide switch is gray off.

The difference is that if you manually "create an account", it will automatically sync, and if you "import", it will disable synchronization.
